### ISSUE TITLE
Icon-only button is always made square

### ISSUE THREAD

**Issue by stanio** [JFormDesigner/FlatLaf#110]
```java
package net.example;

import java.awt.FlowLayout;
import javax.swing.*;

public class ButtonTest extends JFrame {

    ButtonTest() {
        super("Button Test");
        super.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
        super.setLayout(new FlowLayout());

        ImageIcon tallIcon = new ImageIcon(ButtonTest.class.getResource("20x60.png"));
        super.add(new JButton(tallIcon));
        super.add(new JToggleButton(tallIcon));
    }

    public static void main(String[] args) {
        SwingUtilities.invokeLater(() -> {
            try {
                UIManager.setLookAndFeel("com.formdev.flatlaf.FlatLightLaf");
            } catch (Exception e) {
                e.printStackTrace();
            }
            JFrame frame = new ButtonTest();
            frame.pack();
            frame.setVisible(true);
        });
    }

}
```

I guess this likely assumes icon-only buttons are small toolbar buttons, but such are also vertical-label buttons – they should not extend in width just because they are taller.

**Comment by DevCharly** [comment]
Yes, agree with you. This is now changed in master branch.

Please try the latest snapshot `0.36-SNAPSHOT` (or build from master)
https://github.com/JFormDesigner/FlatLaf#snapshots

PS don't forget to "star" FlatLaf here on github 😉

**Comment by stanio** [comment]
Tried with `flatlaf-0.36-20200603.140243-14.jar` – it looks just fine now:

![vertical-button demo](https://user-images.githubusercontent.com/1247730/83714848-9a922280-a634-11ea-8826-437af7431fda.png)

**Comment by stanio** [comment]
I think FlatLaf just rocks!

**Comment by DevCharly** [comment]
> I think FlatLaf just rocks!

That was the plan 😉 
Thanks @stanio 😄

**Comment by DevCharly** [comment]
released in 0.36
